graphix/command.py

```diff
@@ -51,7 +53,30 @@ class M(Command):
     angle: float = 0.0
     s_domain: list[Node] = []
     t_domain: list[Node] = []
-    vop: int = 0
+
+    def clifford(self, clifford: Clifford) -> M:
+        s_domain = self.s_domain
+        t_domain = self.t_domain
+        for gate in clifford.hsz:
+            if gate == graphix.clifford.I:
+                pass
+            elif gate == graphix.clifford.H:
+                t_domain, s_domain = s_domain, t_domain
+            elif gate == graphix.clifford.S:
+                t_domain = s_domain + t_domain
+            elif gate == graphix.clifford.Z:
+                pass
+            else:
```

Comment:
May better to annotate types and use `typing.assert_never`.

Reply:
I don't think types would be currently precise enough to capture this: all the Clifford gates are now in a common class `Clifford`, and even if we introduce a new sub-class `IHSZ`, I don't know how to express the fact that the only instances would be `I`, `H`, `S` and `Z`. Perhaps a dedicated `Enum` would be a solution, but I would leave that for another PR.